lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -299,6 +413,30 @@ export abstract class PerformanceClient implements IPerformanceClient {
      */
     flushMeasurements(measureName: PerformanceEvents, correlationId: string): void {
         this.logger.trace(`PerformanceClient: Performance measurements flushed for ${measureName}`, correlationId);
+
+        /**
+         * Adds all queue time and count measurements for given correlation ID
+         * then deletes queue times for given correlation ID from queueMeasurements map.
+         */
+
+        const queueMeasurementForCorrelationId = this.queueMeasurements.get(correlationId);
+        if (!queueMeasurementForCorrelationId) {
+            this.logger.trace(`PerformanceClient: no queue measurements found for for correlationId: ${correlationId}`);
+        }
+
+        let totalTime = 0;
+        let totalCount = 0;
+        queueMeasurementForCorrelationId?.forEach((measurement) => {
+            if (typeof measurement.queueTime === "number") {
```

Comment:
Is this likely to happen where the queueTime changes type?

Reply:
Good point. I think this was leftover from an earlier version of QueueMeasurement. Removed. Thanks!